Explorer/Assets/DCL/PluginSystem/Global/Web3AuthenticationPlugin.cs

```diff
@@ -39,4 +69,12 @@ private void Login()
             web3Authenticator.LoginAsync(cancellationTokenSource.Token).Forget();
         }
     }
+
+    public struct Web3AuthPluginSettings : IDCLPluginSettings
+    {
+        [field: Header(nameof(Web3AuthenticationPlugin) + "." + nameof(Web3AuthPluginSettings))]
+        [field: Space]
+        [field: SerializeField]
+        public AssetReferenceGameObject AuthScreenPrefab { get; private set; }
```

Comment:
To reference a component on a prefab use a derivative from `ComponentReferernceT` like:

```
[Serializable]
        public class CharacterObjectRef : ComponentReference<CharacterObject>
        {
            public CharacterObjectRef(string guid) : base(guid) { }
        }
```
Thus, you guarantee that component exists on the prefab

Reply:
Hey, this change introduces an issue. I cannot use:
```
AuthenticationScreenView authScreenPrefab = (await assetsProvisioner.ProvideMainAssetAsync(settings.AuthScreenPrefab, ct: ct))
                                                       .Value.GetComponent<AuthenticationScreenView>();
ControllerBase<AuthenticationScreenView, ControllerNoData>.ViewFactoryMethod? authScreenFactory = AuthenticationScreenController.CreateLazily(authScreenPrefab, null);
```

Instead im forced to call:
```
AuthenticationScreenView authScreenPrefab = (await assetsProvisioner.ProvideInstanceAsync(settings.AuthScreenPrefab, ct: ct)).Value;
```

This provokes the prefab to be instantiated at the start of the app but it should be lazy when call:
```
mvcManager.ShowAsync(AuthenticationScreenController.IssueCommand()).Forget();
```

Reply:
There is no natural limitation for that: I just added an overload to expose such a capability https://github.com/decentraland/unity-explorer/pull/226/commits/adae8830a051060fa66a2ac4287ad809e6ba04cd